modules/admin/admin-ui/src/main/resources/web/admin/common/js/form/inputtype/text/HtmlArea.ts

```diff
@@ -237,6 +241,17 @@ module api.form.inputtype.text {
             });
         }
 
```

Comment:
Replace the two methods below with one called ``appendButtonToBottomBar``where you pass the identifying class of the button to move.